### PR TITLE
Prevent content bleeding in Modal

### PR DESCRIPTION
**Pull request by robinandeer**
Fix issues in Safari by using mask-image.
Center align cross icon ONCE AND FOR ALL!
Clean up export of Cross SVG icon.

Reference for Safari overflow bleed fix: https://gist.github.com/ayamflow/b602ab436ac9f05660d9c15190f4fd7b

### No more bleeding in Safari 🎉 

Screenshot 2021-07-07 at 10 58 17

### Cross icon is centered in Mobile Safari 🙏 

Simulator Screen Shot - iPhone 12 Pro Max - 2021-07-07 at 11 12 20

### Cross icon stays centered on Desktop Safari and Chrome

Screenshot 2021-07-07 at 11 15 59

Screenshot 2021-07-07 at 11 16 19

### Cross icon behaves the same as before when used elsewhere

Screenshot 2021-07-07 at 11 23 37

### CODE REVIEW

File: src/Components/Actions/AddressAutocompleteAction/Modal.tsx [hunk 1]
```diff
@@ -29,9 +29,11 @@ const StyledModal = styled(motion.div)`
   flex-direction: column;
 
   @media (min-width: 600px) {
+    max-height: 600px;
     border-radius: 8px;
     overflow: hidden;
-    max-height: 600px;
+    /* Prevent visual content from bleeding in Safari */
+    -webkit-mask-image: -webkit-radial-gradient(white, black);
```
Comment (gustaveen) [inline src/Components/Actions/AddressAutocompleteAction/Modal.tsx:36]: Dark magic! Once again proves that Safari is the new IE :D

**Review by gustaveen — APPROVED** [review]
Looks good, great digging! ⭐️

File: src/Components/Icons/Cross.tsx [hunk 2]
```diff
@@ -2,49 +2,12 @@ import * as React from 'react'
 
 export const Cross = () => (
   <svg viewBox="0 0 13 13" version="1.1" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (robinandeer) [reply src/Components/Icons/Cross.tsx:4]: No, I was looking into if I could re-export it from Figma but didn't quite find the icon

I don't think it's possible to just change the view box coordinates here though.

File: src/Components/Actions/AddressAutocompleteAction/Modal.tsx (hunk 1 above)
Comment (robinandeer) [reply src/Components/Actions/AddressAutocompleteAction/Modal.tsx:36]: Ughhh... at least this solution is easy to isolate to only affect Safari 😅

**Review by gustaveen — APPROVED** [review]

**Pull request merged**